onnx/test/reference_evaluator_ml_test.py

```diff
@@ -12,6 +12,7 @@
 from onnx.checker import check_model
 from onnx.defs import onnx_opset_version
 from onnx.helper import (
+    ORT_MAX_IR_SUPPORTED_VERSION,
```

Comment:
I think we should avoid adding constants related to onnxruntime in the code. What about the other runtime? This information should be placed in onnxruntime.

Reply:
Same thought. I am not sure whether current ONNX Runtime's API provides that and so I created an issue to track this: https://github.com/microsoft/onnxruntime/issues/14932.